src/components/dashboard/FeaturedApps/FeaturedApps.tsx

```diff
@@ -53,17 +45,28 @@ export const FeaturedApps = ({ stackedLayout }: { stackedLayout: boolean }): Rea
         </Typography>
         <WidgetBody>
           <Grid container spacing={3} height={1}>
-            {featuredApps?.map((app) => (
-              <Grid item xs={12} md={stackedLayout ? 12 : 6} key={app.id}>
-                <NextLink
-                  passHref
-                  href={{ pathname: AppRoutes.apps.open, query: { ...router.query, appUrl: app.url } }}
-                  onClick={enableWc && isWalletConnectSafeApp(app.url) ? onWcWidgetClick : undefined}
-                >
-                  <FeaturedAppCard app={app} />
+            {txBuilder?.app && (
+              <Grid item xs={12} md={stackedLayout ? 12 : 6}>
+                <NextLink passHref href={txBuilder?.link}>
+                  <FeaturedAppCard
+                    name={txBuilder.app.name}
+                    description={txBuilder.app.description}
+                    iconUrl={txBuilder.app.iconUrl}
+                  />
                 </NextLink>
               </Grid>
-            ))}
+            )}
+            {enableWc && (
+              <Grid item xs={12} md={stackedLayout ? 12 : 6}>
+                <a onClick={onWcWidgetClick}>
```

Comment:
Without href the cursor is not a pointer even though the whole card is clickable. I suggest we add some hover effect. We could also do `href="#"` and `e.preventDefault()` onClick.